website/docs/r/security_center_vm_server_vulnerability_assessment.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+subcategory: "Security Center"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_security_center_vm_server_vulnerability_assessment"
+description: |-
+  Manages an Azure Vulnerability Assessment (Qualys) to a VM.
+---
+
+# azurerm_security_center_vm_server_vulnerability_assessment
+
+Manages an Azure Server Vulnerability Assessment (Qualys) to a VM.
+
+-> **NOTE** Azure Defender has to be enabled on the subscription in order for this resource to work.
+See this [documentation](https://docs.microsoft.com/en-us/azure/security-center/security-center-get-started) to get started.
```

Comment:
(as above) removing the language here will redirect to the article in the users locale if available

```suggestion
See this [documentation](https://docs.microsoft.com/azure/security-center/security-center-get-started) to get started.
```